src/rgw/radosgw-admin/radosgw-admin.cc

```diff
@@ -4112,6 +4113,8 @@ int main(int argc, const char **argv)
       // do nothing
     } else if (ceph_argparse_binary_flag(args, i, &yes_i_really_mean_it, NULL, "--yes-i-really-mean-it", (char*)NULL)) {
       // do nothing
+    } else if (ceph_argparse_binary_flag(args, i, &tech_preview, NULL, "--tech-preview", (char*)NULL)) {
```

Comment:
why do we need this flag?
we already require the `yes-i-really-mean-it` flag.
also, AFAIK, `tech preview` is a downstream term. on `main` everything is a `tech preview`

Reply:
I want to have this PR backported to tentacle to allow customers to try full dedup on test setup

Reply:
i still dont see why we need another flag. especially that it is used a a downstream term ('tentacle" is a stable upstream version)